nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/api/ProcessGroupResource.java

```diff
@@ -552,41 +553,43 @@ public Response deleteVariableRegistryUpdateRequest(
     public Response updateVariableRegistry(
         @Context final HttpServletRequest httpServletRequest,
         @ApiParam(value = "The process group id.", required = true) @PathParam("id") final String groupId,
-        @ApiParam(value = "The process group configuration details.", required = true) final VariableRegistryEntity requestEntity) {
+        @ApiParam(value = "The process group configuration details.", required = true) final VariableRegistryEntity requestVariableRegistryEntity) {
```

Comment:
Should the `ApiParam` value be updated to reflect that this is VR-specific as opposed to generic PG configuration?

Reply:
Yes good catch.